README.md

```diff
@@ -20,7 +20,7 @@
 
 Argo CD is a declarative, GitOps continuous delivery tool for Kubernetes.
 
-![Argo CD UI](docs/assets/argocd-ui.gif)
+![Argo CD UI](docs/assets/argocd-ui-updated.gif)
```

Comment:
I know size is a consideration, but can we go slightly higher on the resolution? I think the old gif is significantly more detailed.